ci/cron/src/Main.hs

```diff
@@ -93,24 +93,21 @@ build_and_push temp versions = do
             putStrLn $ "Building " <> show version <> "..."
             build version
             putStrLn $ "Pushing " <> show version <> " to S3 (as subfolder)..."
-            push (temp </> show version) $ show version
+            push version
             putStrLn "Done.")
     where
         restore_sha io =
-            Control.Exception.bracket (init <$> shell "git rev-parse HEAD")
+            Control.Exception.bracket (init <$> shell "git rev-parse --abbrev-ref HEAD")
```

Comment:
This fixes a small issue: previously, after running the docs cron (which only CI does and always in ephemeral nodes, so the notion of "after" is a bit flimsy, but still), the cwd would end up with the right commit checked out but in detached HEAD state. By using the branch name here, we avoid that.

On the other hand, if run without a branch to start with, this resolves to `HEAD`, so maybe it's not that much better. 🤔